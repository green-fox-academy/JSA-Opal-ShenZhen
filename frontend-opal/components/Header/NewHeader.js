import React, { Component } from 'react';
import { Container, Header, Item, Input, Icon, Left, Body, Right, Button, Title, Text } from 'native-base';

const NewHeader = () => {
    return (
        <Container>
            <Header searchBar rounded>
                <Icon name="search"></Icon>
                <Item>
                    <Icon name="ios-search" />
                    <Input placeholder="Search" />
                    <Icon name="ios-people" />
                </Item>

                <Button transparent>
                    <Text>Search</Text>
                </Button>
            </Header>
        </Container>
    )
}

export default NewHeader;