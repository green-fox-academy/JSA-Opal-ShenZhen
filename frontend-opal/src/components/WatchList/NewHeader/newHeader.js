import React from 'react';
import { Header, Left, Body, Right, Button, Icon, Title } from 'native-base';

export default function NewHeader() {
  return (
    <Header>
      <Left>
        <Button transparent>
          <Icon name="menu" />
        </Button>
      </Left>
      <Body>
        <Title>Header</Title>
      </Body>
      <Right>
        <Button transparent>
          <Icon name="search" />
        </Button>
      </Right>
    </Header>
  );
}
