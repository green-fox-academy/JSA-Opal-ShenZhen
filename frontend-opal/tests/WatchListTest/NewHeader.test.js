import React from 'react';
import renderer from 'react-test-renderer';

import NewHeader from '../../src/components/WatchList/NewHeader/newHeader';

describe('<NewHeader />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<NewHeader />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
