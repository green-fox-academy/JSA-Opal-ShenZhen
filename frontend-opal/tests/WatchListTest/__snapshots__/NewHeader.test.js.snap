// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NewHeader /> renders correctly 1`] = `
<View
  onLayout={[Function]}
>
  <View
    style={
      Object {
        "backgroundColor": "#F8F8F8",
        "borderBottomColor": "#a7a6ab",
        "borderBottomWidth": 0.5,
        "elevation": 3,
        "flexDirection": "row",
        "height": 64,
        "justifyContent": "center",
        "left": 0,
        "paddingLeft": 6,
        "paddingRight": 10,
        "paddingTop": 18,
        "right": 0,
        "shadowColor": undefined,
        "shadowOffset": undefined,
        "shadowOpacity": undefined,
        "shadowRadius": undefined,
        "top": 0,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "flex-start",
          "alignSelf": "center",
          "flex": 1,
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": null,
            "backgroundColor": "transparent",
            "borderBottomWidth": null,
            "borderColor": null,
            "borderLeftWidth": null,
            "borderRadius": 5,
            "borderRightWidth": null,
            "borderTopWidth": null,
            "borderWidth": undefined,
            "elevation": 0,
            "flexDirection": "row",
            "height": 45,
            "justifyContent": "space-between",
            "marginLeft": -3,
            "opacity": 1,
            "paddingBottom": 6,
            "paddingLeft": 4,
            "paddingRight": 6,
            "paddingTop": 6,
            "shadowColor": null,
            "shadowOffset": null,
            "shadowOpacity": null,
            "shadowRadius": null,
          }
        }
      >
        <Text />
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "alignSelf": "center",
          "flex": 1,
        }
      }
    >
      <Text
        numberOfLines={1}
        style={
          Object {
            "color": "#000",
            "fontFamily": "System",
            "fontSize": 17,
            "fontWeight": "700",
            "marginLeft": undefined,
            "paddingLeft": 4,
            "paddingTop": 1,
            "textAlign": "center",
          }
        }
      >
        Header
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "flex-end",
          "alignSelf": "center",
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "flex-end",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": null,
            "backgroundColor": "transparent",
            "borderBottomWidth": null,
            "borderColor": null,
            "borderLeftWidth": null,
            "borderRadius": 50,
            "borderRightWidth": null,
            "borderTopWidth": null,
            "borderWidth": undefined,
            "elevation": 0,
            "flexDirection": "row",
            "height": 45,
            "justifyContent": "space-between",
            "marginRight": -9,
            "opacity": 1,
            "paddingBottom": 6,
            "paddingLeft": 15,
            "paddingRight": 15,
            "paddingTop": 6,
            "shadowColor": null,
            "shadowOffset": null,
            "shadowOpacity": null,
            "shadowRadius": null,
          }
        }
      >
        <Text />
      </View>
    </View>
  </View>
</View>
`;
